app/javascript/activities.js

```
@@ -1,16 +1,13 @@
 import Vue from 'vue/dist/vue.esm';
-import TurbolinksAdapter from 'vue-turbolinks';
-import VueResource from 'vue-resource';
 
-Vue.use(TurbolinksAdapter);
 Vue.use(VueResource);
```

ESLint

[error] 'VueResource' is not defined.

(no-undef)